src/shellstrings.c

```diff
@@ -7,88 +7,331 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <stdbool.h>
 #include <string.h>
 #include "shellstrings.h"
+#include "log.c/src/log.h"
 
 /* See shellstrings.h */
-void shell_intro() {
-    printf("\nPlease load a file to begin. Additionally, select an output mode and/or choose a dictionary before running the program.\n");
-
-    printf("f [/path/file.txt]           : open file\n");
-    printf("d [/path/dictionary.txt]     : input custom dictionary file\n");
-    printf("m [number]                   : switch output mode - quiet (1) verbose (2) or interactive (3) \n");
-    printf("h                            : help\n");
-    printf("q                            : quit program\n");
+void shell_prompt(bool *color) {
```

Comment:
Why is this a `bool *` and not just a `bool`?

Reply:
changed